packages/next/src/client/components/react-dev-overlay/_experimental/internal/container/runtime-error/use-error-hook.ts

```diff
@@ -21,17 +21,13 @@ export type SupportedErrorEvent = {
 
 function getErrorSignature(ev: SupportedErrorEvent): string {
```

Comment:
Once the `switch` is not exhaustive, TypeScript will error here because that would change the return type to `string | void`.